src/main/java/amazon/AllSearchResultPage.java

```diff
@@ -34,11 +34,11 @@ public boolean textVisibilityWhenClickOnBestSellerButton() {
         int randomNumber=random.nextInt(1, listItemWhereWriteBestSeller.size());
         listItemWhereWriteBestSeller.get(randomNumber).click();
         boolean textContains = textListWhenClickOnBestSellerButton.get(randomNumber).getText().contains("in");
+        textVisibility=textContains;
         if (textVisibility == false) {
-            return textVisibility;
+            return false;
```

Comment:
The method is too long and complicated. Read one more time task requirement and simplify the method.